docs/topics/booster-deploy-ocp.adoc

```diff
@@ -0,0 +1,5 @@
+= Deploying your Booster to {OpenShiftContainerPlatform}
+
+IMPORTANT: In order to deploy a booster to {OpenShiftContainerPlatform}, you must first create it the {launcher} application in xref:oso-create-booster[{OpenShiftOnline}] or xref:osl-create-booster[a {OpenShiftLocal}].
```

Comment:
Sorry for my inability to find all issues in one go :-), but there's a word missing: "you must first create it the {launcher} application in {OpenShiftOnline}a {OpenShiftLocal}". It should (probably?) read "you must first create it _using_ the {launcher} application ...".

Reply:
Also I'm not sure in which context this text is used, but I guess it assumes that an existing Launcher (in OSO or Minishift) is used to create a .zip, but there's also an option of deploying Launcher to OCP just like you can deploy it to Minishift. Then, you can use it to directly launch a booster in the OCP instance.